File: 8KYU/Find the first non-consecutive number/solution.java
class FirstNonConsecutive {
    static Integer find(final int[] arr) {
       for(var i = 0; i < arr.length-1; i++) {
          if(arr[i+1] - arr[i] > 1) {
            return arr[i+1];
          }
       }
       return null;
    }   
 }
 

File: 8KYU/Find the first non-consecutive number/solution.rs
fn first_non_consecutive(arr: &Vec<i32>) -> Option<i32> {
    for i in 0..arr.len() -1 {
        if arr[i+1] - arr[i] > 1 {
            return Some(arr[i+1]);
        }
      }
      return None;
}